src/renderer/captureTool/js/capture.js

```diff
@@ -146,7 +146,20 @@
             // Prefs Set Name to Save the captured settings to. This is either
             // what is typed in the input for saving to a new set, or the `selectedPrefsSetName`
             // if an existing set is chosen.
-            prefsSetName: "MyCapture"
+            prefsSetName: "MyCapture",
+
+            // Value to keep track of if we are currently saving the capture preferences.
+            //
+            // Because we depend on the same model events as the rest of the QSS (preferencesSavedSuccess, and
+            // preferencesSavedError) to determine if preferences are successfully saved, we set the value of
+            // this model property to determine if we are between the second to last and final screen of the capture
+            // tool. Saving occurs between this final dialog transition. If this value is false we can safely ignore the
+            // event. If we don't track capture saving state we'll jump to the capture saved confirmation screen if
```

Comment:
A better approach would instantiate a dynamic "capture" component holding the event listener only for the duration of the interaction, rather than having the capture component present all the time. I won't expect this for this review, this is just a note for future "best practices" : P

Reply:
Thanks for this edifying glimpse into... THE FUTURE!